Makefile

```diff
@@ -106,6 +106,11 @@ else
 endif
 endif
 
+# lapack may be a separate library
+ifeq ($(USE_BLAS), openblas)
+        LDFLAGS += -llapack
```

Comment:
I'm currently tracing the details. It is available, but at least on ubuntu I have to wget get it as a separate package. Guess I will have it worked out by tomorrow.
So far:
- The only consistent way to use lapack in MXNet by includes is to interface with the Fortran code. openblas has lapacke.h, but ATLAS and "apple" (accelarator framework) use clapack.h. The latter have completely different signatures, ATLAS is a proper C-wrapper while "apple" are just C-declarations matching the Fortran signatures. So it is easier to  interface with the Fortran code in all cases. 
- We have some builds that just use "blas" or even "NONE". These are builds for docs and also the mxnet-p-r (see failure below).  My plan is to support such builds (which are not intended to generate an executable that actually runs) by passing a -DMXNET_USE_LAPACK (in all other builds). If this is not present, then our lapack-wrappers will be compiled  into a stub that will create an error at runtime. I don't want to completely skip compiling operators that use lapack as we may get a lot more lapack-based functionalities in the future and it would be cumbersome to handle them all specially in such builds. 

What do you think?